### PR TITLE
[FEAT] create card from payment nonce generated by Accept.js

### PR DESCRIPTION
**Pull request by ninjarails**

### CODE REVIEW

File: app/models/accounting/payment.rb [hunk 1]
```diff
@@ -111,7 +111,7 @@ def expiration_date
 
       def format_data
         # Ensure the year is 4 digit representation
-        self.year = '20' + year.to_s[-2..-1].to_s
+        self.year = '20' + year.to_s[-2..-1].to_s unless year == -1
```
Comment (alexbrinkman) [inline app/models/accounting/payment.rb]: Why would the year be -1?

File: lib/accounting/test/create_card.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
+module Accounting
```
Comment (alexbrinkman) [inline lib/accounting/test/create_card.rb]: This seems to be just to make the tests pass.  Do we need to refactor tests too to reflect the change to Accept.js?

File: app/models/accounting/payment.rb (hunk 1 above)
Comment (ninjarails) [reply app/models/accounting/payment.rb]: https://github.com/CKDev/accounting/blob/master/app/services/accounting/payment_service.rb#L20

File: lib/accounting/test/create_card.rb (hunk 2 above)
Comment (ninjarails) [reply lib/accounting/test/create_card.rb]: In current Accounting gem we use Accept.js payment nonce but I don't know a way to create payment nonce in RSpec. But we need to create Accounting::Payment in specs to run Service and Hook specs. This class provides the way to create Accounting::Payment without using Accept.js payment nonce but using Payment Card informations.

Comment (ninjarails) [reply lib/accounting/test/create_card.rb]: I think I have done all possible refactoring on Specs at this point.

**Pull request merged**